packages/openapi-framework/src/types.ts

```diff
@@ -15,6 +15,34 @@ export {
   OpenAPIErrorTransformer
 };
 
+export class ConsoleDebugAdapterLogger implements Logger {
+  /**
+   * `console.debug` is just an alias for `.log()`, and we want debug logging to be optional.
+   * This class delegates to `console` and overrides `.debug()` to be a no-op.
+   * @param message
+   * @param optionalParams
+   */
+  public debug(message?: any, ...optionalParams: any[]): void {
+    // no-op
+  }
+
+  public error(message?: any, ...optionalParams: any[]): void {
+    console.error(message, optionalParams);
```

Comment:
i think we need to spread the optionalParams

Reply:
what do you mean, `console.error(message, ...optionalParams);` ?

Reply:
> what do you mean, `console.error(message, ...optionalParams);` ?

yes.  otherwise we're passing an array to the console methods correct?

Reply:
that's correct, now looks better